core/arch/riscv64/emit_utils.c

```diff
@@ -292,6 +292,9 @@ void
 patch_stub(fragment_t *f, cache_pc stub_pc, cache_pc target_pc, cache_pc target_prefix_pc,
```

Comment:
I assume this patch fixes some observed crash? Please add that detail to the PR description.

Reply:
Added the details.